lms/templates/fragment-view-chromeless.html

```diff
@@ -0,0 +1,70 @@
+## mako
+
+<%! main_css = "style-main-v2" %>
+
+<%page expression_filter="h"/>
+<%namespace name='static' file='static_content.html'/>
+
+<%!
+import json
+from django.utils.translation import ugettext as _
+from django.template.defaultfilters import escapejs
+from django.core.urlresolvers import reverse
+from pipeline_mako import render_require_js_path_overrides
+
+from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
+from openedx.core.djangolib.markup import HTML
+%>
+
+<!DOCTYPE html>
+<html lang="${LANGUAGE_CODE}">
+<head dir="${static.dir_rtl()}">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<head>
+    <title></title>
+    <%
+    jsi18n_path = "js/i18n/{language}/djangojs.js".format(language=LANGUAGE_CODE)
+    %>
+    <script type="text/javascript" src="${static.url(jsi18n_path)}"></script>
+
+    <%static:css group='style-vendor'/>
+    <%static:css group='${self.attr.main_css}'/>
+
+    % if not uses_pattern_library:
```

Comment:
A lot of this is borrowed from main.html, in the same way as courseware-chromeless.html does. There does seem to be a good opportunity for a shared template here. I'll see what I can do.